### PR TITLE
Add example issues for testing PR annotation.

### PR DESCRIPTION
**Pull request by mwz**

### CODE REVIEW

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 1]
```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
     }
   }
 
+  def ExampleIssues: Unit = {
```
Comment (mwz) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: test

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Parameterless methods returns unit ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.style.ParameterlessMethodReturnsUnit&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.style.ParameterlessMethodReturnsUnit))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: INFO: Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Method name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 2]
```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
...
+    val isEmpty = List(1).size == 0
+    val b = if (true) true else false
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: INFO: Unnecessary if condition. ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.UnnecessaryIf&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.UnnecessaryIf))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Constant if expression ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.ConstantIf&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.ConstantIf))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: No redundant if expressions ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.RedundantIfChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.RedundantIfChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 3]
```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
...
+    var TWO = 1 + 1
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Var could be val ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MAJOR: No mutable local variables ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.VarLocalChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.VarLocalChecker))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Field name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 4]
```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
...
+    while (true) {
+      println("WOW!")
+    }
+    return ()
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Return ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 5]
```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
     }
   }
 
+  def ExampleIssues: Unit = {
+    val isEmpty = List(1).size == 0
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: INFO: List.size is O(n) ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Avoid Traversable.size == 0, use Traversable.isEmpty instead ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.AvoidSizeEqualsZero&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.AvoidSizeEqualsZero))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Literal passed as argument without name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NamedArgumentChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NamedArgumentChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 6]
```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
...
+    val b = if (true) true else false
+    var TWO = 1 + 1
+    while (true) {
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: While true loop ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: No while loops ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 7]
```diff
@@ -69,6 +69,17 @@ final class ScalaSensor extends Sensor {
     }
   }
 
+  def ExampleIssues(): Unit = {
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: INFO: Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Method name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 8]
```diff
@@ -69,6 +69,17 @@ final class ScalaSensor extends Sensor {
...
+    val isEmpty = List(1).size == 0
+    val b = if (true) true else false
+    var TWO = 1 + 1
+    //test
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: MINOR: Space after the start of the comment ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 9]
```diff
@@ -69,6 +69,17 @@ final class ScalaSensor extends Sensor {
...
+    while (true) {
+      println("WOW!")
+    }
+    return ()
+  }
+
   override def describe(descriptor: SensorDescriptor): Unit = {
     descriptor
       .onlyOnLanguage(Scala.LanguageKey)
```
Comment (mwz) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: another test comment

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 10]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
       }
   }
 
+  def ExampleIssues(): Unit = {
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: info Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Method name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.MethodNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 11]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    val isEmpty = List(1).size == 0
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: info List.size is O(n) ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Avoid Traversable.size == 0, use Traversable.isEmpty instead ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.AvoidSizeEqualsZero&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.AvoidSizeEqualsZero))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Literal passed as argument without name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NamedArgumentChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.NamedArgumentChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 12]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    val b = if (true) true else false
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: info Unnecessary if condition. ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.UnnecessaryIf&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.UnnecessaryIf))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Constant if expression ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.ConstantIf&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.ConstantIf))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor No redundant if expressions ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.RedundantIfChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.RedundantIfChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 13]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    var TWO = 1 + 1
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Var could be val ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: major No mutable local variables ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.VarLocalChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.VarLocalChecker))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Field name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 14]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    //test
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Space after the start of the comment ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 15]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+    while (true) {
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor While true loop ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor No while loops ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala [hunk 16]
```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
...
+      println("WOW!")
+    }
+    return ()
```
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Return ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala (hunk 13 above)
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Field name ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.FieldNamesChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala (hunk 14 above)
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Space after the start of the comment ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala (hunk 15 above)
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor While true loop ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.controlflow.WhileTrue))

Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor No while loops ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.WhileChecker))

File: src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala (hunk 16 above)
Comment (sonarscala) [inline src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala]: minor Return ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.ReturnChecker))

**Pull request closed without merging**